### PR TITLE
Updates for jenkins

### PR DESCRIPTION
**Pull request by SRaus**

### CODE REVIEW

File: build-docker.sh [hunk 1]
```diff
@@ -77,23 +77,27 @@ ${DOCKER} build -t pi-gen "${DIR}"
...
+		-v /var/lib/jenkins/workspace/Kuiper/work:/pi-gen/work \
 		--volume "${CONFIG_FILE}":/config:ro \
 		-e "GIT_HASH=${GIT_HASH}" \
 		--volumes-from="${CONTAINER_NAME}" --name "${CONTAINER_NAME}_cont" \
 		pi-gen \
 		bash -e -o pipefail -c "dpkg-reconfigure qemu-user-static &&
-	cd /pi-gen; ./build.sh ${BUILD_OPTS} &&
-	rsync -av work/*/build.log deploy/" &
+		cd /pi-gen; ./build.sh ${BUILD_OPTS} &&
+		rsync -av work/*/build.log deploy/ &&  chown -R $(id -u):$(id -g) ./work/ ./deploy/" &
```
Comment (mirceacaprioru) [inline build-docker.sh:88]: this change is not related to the commit. why are you changind the owner for the work directory?

**Review by mirceacaprioru — CHANGES_REQUESTED** [review]

File: build-docker.sh (hunk 1 above)
Comment (SRaus) [reply build-docker.sh:88]: If I'm not changing the owner the flow won't work. Files from work/ and deploy/ will be owned by 'root' and 'jenkins' user won't have any right to them, not even to copy/export.
When I'm building just stage 6 to 9, I'm adding ./work and ./deploy folder as volumes in docker container, so that the flow will see stages 0-5 as already built. Since Docker is run by jenkins, if files are own by root, the volumes cannot be added. Even if 'jenkins' user will have suddo rights, Jenkins Framework does not allow to run commands as 'sudo'.

Comment (mirceacaprioru) [reply build-docker.sh:88]: we have to test this a bit more. and check sudo and root with an imagine build with this modification

**Comment by SRaus** [comment]
We want to test first if files ownership (necessary by jenkins flow) introduces any bugs.
